src/main.js

```diff
@@ -9,70 +8,94 @@ var btnMakeNewCover = document.querySelector(".make-new-button");
 var btnViewSavedCover = document.querySelector(".view-saved-button");
 var btnHome = document.querySelector(".home-button");
 var btnSaveCover = document.querySelector(".save-cover-button");
-var btnMakeMyBook = document.querySelector(".create-new-book-button");//3rd iteration
+var btnMakeMyBook = document.querySelector(".create-new-book-button");
 var homeView = document.querySelector(".home-view");
 var savedView = document.querySelector(".saved-view");
 var formView = document.querySelector(".form-view");
-var inputCover = document.querySelector(".user-cover");///vars below will be on Jessica's branch
+var inputCover = document.querySelector(".user-cover")
 var inputTitle = document.querySelector(".user-title");
 var inputDescriptor1 = document.querySelector(".user-desc1");
 var inputDescriptor2 = document.querySelector(".user-desc2");
 var form = document.querySelector("form");
 
 var savedCovers = [
-  new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
+  new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
 
+// Event listeners
 window.addEventListener("load", createRandomCover);
 btnMakeNewCover.addEventListener("click", showForm);
 btnRandomCover.addEventListener("click", createRandomCover);
 btnViewSavedCover.addEventListener("click", showSaved);
 btnHome.addEventListener("click", showHome);
+btnMakeMyBook.addEventListener("click", createNewCover);
 form.addEventListener('submit', handleForm);
 
+//Prevents page reload upon submitting form
 function handleForm(event) { event.preventDefault(); }
 
+//Hides home view and associated buttons, displays home button
 function leaveHome() {
-  homeView.classList.add("hidden");
-  btnSaveCover.classList.add("hidden");
-  btnRandomCover.classList.add("hidden");
+  homeView.classList.add("hidden");
+  btnSaveCover.classList.add("hidden");
+  btnRandomCover.classList.add("hidden");
   btnHome.classList.remove("hidden");
 }
```

Comment:
so i tend to not use semi-colons anywhere except for where they're actually required but if you're wanting to use them, make sure you have them everywhere they need to be. for instance, there should be one after every closing curly brace :)

Reply:
Thank you for pointing that out!